.github/workflows/antora.yml

```diff
@@ -15,10 +15,18 @@ on:
         type: boolean
         required: false
         default: false
+      fork-schedule:
```

Comment:
I'm not sure that it's a good name. @mkurz do you have other idea for name? 🤔

Reply:
Hmm... not sure, what about `run-scheduled-in-forks`? I think there needs to be a work like `run` or `execute`.

Reply:
Good, I like it. 👍